docs/EmbededApp.md

```diff
@@ -0,0 +1,141 @@
+---
+id: embeded-app
+title: Use React Native in Existing iOS App
+layout: docs
+category: Guides
+permalink: docs/embeded-app.html
+next: activityindicatorios
+---
+
+## Install React Native Using Cocoapods
+
+[Cocoapods](http://cocoapods.org/) is a package management tool for iOS/Mac development. We need to use it to download React Native. If you haven't install Cocoapods yet, checkout [this tutorial](http://guides.cocoapods.org/using/getting-started.html).
+
+When you are ready to work with Cocoapods, add the following line to `Podfile`. If you don't have one, then create it under the root directory of your project.
+
+```
+pod 'React'
+pod 'React/RCTText'  
+# Add any subspecs you want to use in your project
+```
+
+Remember to install all subspecs you need. The `<Text>` element cannot be used without `pod 'React/RCTText'`.
+  
+Then install pods via shell
+
+```
+$ pod install --verbose
+```
+  
+The installation process also requires [Node.js](http://nodejs.org).
+
+## Create Your React Native App
+
+First, enter React Native's pod root directory and create **index.ios.js** inside a directory `ReactComponent`.
+
+```
+$ cd Pods/React
+$ mkdir ReactComponent
+$ touch index.ios.js
+```
+  
+Copy & paste following starter code for **index.ios.js**.
+
+```
+var React = require('react-native');
+
+var {
+  Text,
+  View
+} = React;
+
+var styles = React.StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: 'red'
+  }
+});
+
+class SimpleApp extends React.Component {
+  render() {
+    return <View style={styles.container}>
+        <Text>This is a simple application.</Text>
+      </View>;
+  }
+}
+
+React.AppRegistry.registerComponent('SimpleApp', () => SimpleApp);
+```
+  
+`SimpleApp` will be your **module name**, which will be used later on.
+
+## Add Container View To Your App
+
+You should now add container view for React Native component. It can be any **UIView** in your app.
+
+![Container view example](http://7qn8cx.com1.z0.glb.clouddn.com/c37deb761ce048225989ece06cf3d185061c511d-74d3e6b285e1b5cd93153c39fca63c63838b8dbf.png?imageView2/2/h/400/w/400/q/85)
```

Comment:
Can you add this image in the repo (website/src/react-native/img) this way it won't 404 if this url goes down.